### DISCUSSION TITLE
How do I correctly export to other note taking apps?

### DISCUSSION THREAD

**Discussion by Aitester101** [TriliumNext/Trilium#4703] [Q&A] [answered]
Hi there,

Loving it so far, it offers everything I need in a note-taking app and more. My only concern lies in transferring notes to other platforms.

Here's the situation: I currently use multiple note-taking apps due to pre-existing workflows and some syncing challenges across devices with Trilium. While Trilium is fantastic, a smoother syncing solution would be incredible.

This reliance on multiple apps stems from the difficulty in seamlessly syncing Trilium notes across various platforms, particularly mobile.

I understand the option to export a tree of notes via the context menu exists. However, upon import to other apps (like Obsidian), formatting seems to break. So, am I doing something wrong, or what app can correctly read the Trilium notes?

This is crucial for both backups and overall reliability. As someone who would like to rely on Trilium for all my note-taking needs, ensuring my notes can be accessed reliably in other apps, or at least get Trilium to sync everywhere reliably provides peace of mind in case of unforeseen circumstances.

Thank you for your time and consideration.

**Comment by meichthys** [comment, marked as answer]
When you export notes, you would need to choose Markdown as the export format in order to make it compatible with other applications that don't support html format. You will loose some formatting since Trilium uses the CKeditor which is an HTML editor. For a discussion about this, check out the TriliumNext fork which is discussing the possibility of using different editors (perhaps a markdown-only editor) for those who want to preserve formatting better for exporting to other applications.
